README.md

```diff
@@ -0,0 +1,335 @@
+# virus-scanner-service
+
+Microservice to perform a virus scan on uploaded files.
+
+This service listens for delta notifications about new files and scans
+those for viruses.
+
+[![Project Status: WIP – Initial development is in progress](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
+
+## Getting started
+
+Prerequisites:
+- [file-service](https://github.com/mu-semtech/file-service)
+- [mu-authorization](https://github.com/mu-semtech/mu-authorization)
+- [delta-notifier](https://github.com/mu-semtech/delta-notifier)
+
+### Adding the service to your stack
+
+Add the following snippet to your `docker-compose.yml` to include this
+service in your project.
+
+```yaml
+version: '3.4'
+
+services:
+  virus-scanner:
+    build: https://github.com/peternowee/virus-scanner-service.git#v0.0.8
+    links:
+      - database:database
+    environment:
+      VIRUS_SCANNER_CLAMD_USER: # "root"
```

Comment:
The user needs to be root? or it's default to 'root'?

Reply:
Hopefully clarified by commit 1d8ed0536903e7c4ca7daef38bc7bda52b52e77f .